src/lib.rs

```diff
@@ -174,10 +185,20 @@ impl Universe {
         self.width
     }
 
+    pub fn set_width(&mut self, width: u32) {
```

Comment:
Could you also add the doc comments that exist in the testing chapter of the book? Will merge once those are here :)